config/routes.rb

```diff
@@ -1,4 +1,14 @@
 Rails.application.routes.draw do
+  get 'users/index'
+  get 'users/show'
+  devise_for :users, controllers: {
+    registrations: "users/registrations"
+  }
+
+  resources :users, :only => [:index, :show]
```

Comment:
すごく細かいですし個人的な好みなので、無視してしまってかまわないですが、

```suggestion
  resources :users, only: [:index, :show]
```

の方がスッキリするかなーと思います